### PR TITLE
feat: create user signin controller

### PR DESCRIPTION
**Pull request by SidnySilva**
Created signin function .
Inserted JWT lib.
Inserted password space and class methods in UserModel.

### CODE REVIEW

File: app/controllers/user_controller.py [hunk 1]
```diff
@@ -1,18 +1,40 @@
+from http import HTTPStatus
+
+from app.configs.database import db
+from app.models.user_model import UserModel
+from flask import jsonify, request, session
+from flask_jwt_extended import (create_access_token, get_jwt_identity,
+                                jwt_required)
+from sqlalchemy.orm import Session
+
+
```
Comment (marcelosj3) [inline app/controllers/user_controller.py]: Use `isort .` and then `black .` in the terminal, we might run into some conflicts regarding the imports at some point if we don't do that.

File: app/models/message_model.py [hunk 2]
```diff
@@ -19,4 +18,4 @@ class MessageModel(db.Model):
     text = Column(String, nullable=False)
     date = Column(DateTime, nullable=False)
 
-    users = db.relationship("UserModel", secondary=users_messages, backref="messages")
+    # users = db.relationship("UserModel", secondary=users_messages, backref="messages")
```
Comment (marcelosj3) [inline app/models/message_model.py]: You can delete that, i ran into the same error, the backref property already created the "messages" in the other table, i might be wrong, but if we wanna keep that in both tables, we can use back_populates instead.

**Review by marcelosj3 — CHANGES_REQUESTED** [review]

File: app/models/message_model.py (hunk 2 above)
Comment (penguinuux) [reply app/models/message_model.py]: I already fix it in my last pull request. You can delete it.

File: app/controllers/user_controller.py (hunk 1 above)
Comment (penguinuux) [reply app/controllers/user_controller.py]: Agreed!

File: migrations/versions/f06d8f324633_adding_password_to_user.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+"""adding password to user
```
Comment (penguinuux) [inline migrations/versions/f06d8f324633_adding_password_to_user.py]: Need to improve the description of what was changed.

**Review by penguinuux — CHANGES_REQUESTED** [review]

File: app/controllers/user_controller.py (hunk 1 above)
Comment (penguinuux) [reply app/controllers/user_controller.py]: Done.

File: app/models/message_model.py (hunk 2 above)
Comment (penguinuux) [reply app/models/message_model.py]: Done.

File: migrations/versions/f06d8f324633_adding_password_to_user.py (hunk 3 above)
Comment (penguinuux) [reply migrations/versions/f06d8f324633_adding_password_to_user.py]: Done.

**Review by marcelosj3 — APPROVED** [review]

**Review by penguinuux — APPROVED** [review]

**Pull request merged**